tests/trainer/test_trainer.py

```diff
@@ -3451,3 +3456,103 @@ def test_hyperparameter_search_backends(self):
             list(ALL_HYPERPARAMETER_SEARCH_BACKENDS.keys()),
             list(HPSearchBackend),
         )
+
+
+@require_peft
+@require_torch
+class TrainerPeftTest(unittest.TestCase, TrainerIntegrationCommon):
+    def get_regression_adapter_trainer(
+        self,
+        keep_report_to=False,
+        adapter_name="default",
+        adapter_args=None,
+        **kwargs,
+    ):
+        # from transformers import OPTForCausalLM
+        base_model_id = "hf-internal-testing/tiny-random-OPTForCausalLM"
+        pretrained_base_model = AutoModelForCausalLM.from_pretrained(base_model_id).to(torch_device)
+        tokenizer = AutoTokenizer.from_pretrained(base_model_id)
+
+        def m(sample):
```

Comment:
Let's use a better name here

```suggestion
        def tokenize_sample(sample):
```